mmm_eval/adapters/meridian.py

```diff
@@ -532,9 +532,10 @@ def _predict_on_all_data(self) -> np.ndarray:
         # shape (n_chains, n_draws, n_times)
         preds_tensor = self.analyzer.expected_outcome(aggregate_geos=True, aggregate_times=False, use_kpi=True)
         posterior_mean = np.mean(preds_tensor, axis=(0, 1))
-        return posterior_mean
+        posterior_distribution = np.array(preds_tensor).reshape(-1, preds_tensor.shape[-1])
+        return posterior_mean, posterior_distribution
```

Comment:
thought: might be nicer to just return the distribution here and compute the mean downstream